mg_incident/ticket/admin_views.py

```diff
@@ -1,20 +1,51 @@
 from flask_admin.contrib.sqla import ModelView
-
+from flask_admin.contrib.sqla.filters import BaseSQLAFilter
 from mg_incident import db, admin
 from mg_incident.auth import UserRequiredMixin
 from mg_incident.ticket import models
 
 
+class FilterCreatedBy(BaseSQLAFilter):
```

Comment:
Just for ToDo?

Reply:
For futute-future-future TODO)) 
filter ok by `column_filters = ('created_by.username', 'assigned_by.username', 'assigned_to.username',)`